src/Microsoft.Health.Fhir.Core/Features/Operations/Reindex/ReindexJobTask.cs

```diff
@@ -173,10 +178,42 @@ private async Task<bool> TryPopulateNewJobFields()
         {
             // Build query based on new search params
             // Find supported, but not yet searchable params
-            var notYetIndexedParams = _supportedSearchParameterDefinitionManager.GetSearchParametersRequiringReindexing();
+            var possibleNotYetIndexedParams = _supportedSearchParameterDefinitionManager.GetSearchParametersRequiringReindexing();
+            _notYetIndexedParams = new List<SearchParameterInfo>();
```

Comment:
>_notYetIndexedParams [](http://example.com/codeflow?start=12&length=20)

It looks like it's been used only inside `TryPopulateNewJobFields` function. Maybe it should be local variable?

Reply:
Correct, changed to local.